### PR TITLE
Clean up figure usage

### PR DESCRIPTION
**Pull request by KristofKoch**
**Description of proposed changes**

Changed formatting of usage message as per #5341.

### CODE REVIEW

**Comment by KristofKoch** [comment]
@meghanrjones my apologies but it appears my workflow is flawed. For some reason I added five instead of only the last commit to this branch. I don't know where my error is:

````
git checkout -b figure-usage
````
edit `figure.c`
````
git add src/figure.c
git commit
git push KristofKoch figure-usage
````
Would you kindly point out my mistake?

**Comment by meghanrjones** [comment]
Before `git checkout -b figure-usage`, do `git checkout master` and `git pull` in order to create a branch off the master branch with the latest changes included. Your current workflow is creating braches off whichever branch you were on, in this case likely `end-usage`.

**Comment by meghanrjones** [comment]
Since the PR for end is now merged, try merging master into this branch to see if that makes the diff only show the changes for figure:
```
git checkout master
git pull
git checkout figure-usage
git merge master
git push KristofKoch figure-usage
```

File: src/figure.c [hunk 1]
```diff
@@ -38,28 +38,32 @@
 static int usage (struct GMTAPI_CTRL *API, int level) {
 	const char *name = gmt_show_name_and_purpose (API, THIS_MODULE_LIB, THIS_MODULE_CLASSIC_NAME, THIS_MODULE_PURPOSE);
 	if (level == GMT_MODULE_PURPOSE) return (GMT_NOERROR);
-	GMT_Message (API, GMT_TIME_NONE, "usage: %s <prefix> [<formats>] [<psconvertoptions] [%s]\n\n", name, GMT_V_OPT);
+	GMT_Usage (API, 0, "usage: %s <prefix> [<formats>] [<psconvertoptions] [%s]\n\n", name, GMT_V_OPT);
```
Comment (meghanrjones) [inline src/figure.c:41]:
```suggestion
	GMT_Usage (API, 0, "usage: %s <prefix> [<formats>] [<psconvertoptions>] [%s]\n\n", name, GMT_V_OPT);
```

**Review by meghanrjones — APPROVED** [review]

**Comment by meghanrjones** [comment]
@KristofKoch, when you are ready to squash and merge this PR, you can delete the list of commits in the larger box to clean up the git log since those changes are recorded in other PRs. So it should look like this:

Screen Shot 2021-06-28 at 3 51 17 PM

**Pull request merged**